scanner/Makefile

```diff
@@ -82,17 +84,22 @@ scanner-go-build-local:
 	$(SILENT)echo "+ $@"
 	$(SILENT)$(MAKE) GOOS=${HOST_OS} scanner-go-build
 
+OSSLS_NOTICE_DEP := ossls-notice
+ifdef CI
+	OSSLS_NOTICE_DEP := ossls-notice-no-download
+endif
 .PHONY: scanner-image
-scanner-image: scanner-go-build copy-scripts ossls-notice
+scanner-image: scanner-go-build copy-scripts $(OSSLS_NOTICE_DEP)
+scanner-image:
```

Comment:
What is the significance of having `scanner-image` repeated twice here?

Reply:
Nice catch, I believe it is a remnant of another way of handling the 

```
OSSLS_NOTICE_DEP := ossls-notice
ifdef CI
	OSSLS_NOTICE_DEP := ossls-notice-no-download
endif
```

before I settled on this